Bottom: 2f4c325a218ef5d4880d58ee813efbd28f4fff3a
Top:    cf24655a2795fa86d381c13e7f96cede6ef8aeb3
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-15 16:42:09 -0700

Refresh of test

---
diff --git a/lldb/packages/Python/lldbsuite/test/commands/command/script/py_import b/lldb/packages/Python/lldbsuite/test/commands/command/script/py_import
index dd95d355255..1026fd58f65 100644
--- a/lldb/packages/Python/lldbsuite/test/commands/command/script/py_import
+++ b/lldb/packages/Python/lldbsuite/test/commands/command/script/py_import
@@ -26,4 +26,4 @@ command script add -f callables.FooBarObj.ifoobar ifoobar
 
 command script add -f callables.FooBar.sfoobar4 sfoobar4
 command script add -f callables.FooBar.cfoobar4 cfoobar4
-command script add -f callables.FooBarrObj.ifoobar4 ifoobar4
+command script add -f callables.FooBarObj.ifoobar4 ifoobar4
